packages/avatar/src/react/index.js

```diff
@@ -19,13 +19,23 @@ const styles = {
 
 const Avatar = React.forwardRef((props, ref) => {
   const [imageState, setImageState] = React.useState('loading')
```

Comment:
Now that these states for "loading", "error", and other? are being exposed, I suggest that we export them as a named enum.  This is something that would usually go in vars.  

/me. Now that we do js/ a lot, I wonder if we don't drop "vars/" for "js/" these days.